### PR TITLE
Update governance web documentation

### PR DESCRIPTION
**Pull request by afshin**
This PR updates the governance web pages to reorganize the information and update the information to reflect the governance transition.

### CODE REVIEW

**Comment by jasongrout** [comment]
Darian, I did a major refactor of the table of contents and updated various docs to try to get us in shape for the transition. What do you think?

File: bootstrapping_subproject_councils.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
-# Bootstrapping Subproject Councils
+# Transition: Bootstrapping Subproject Councils
+
+:::{note}
+This document outlines how subprojects initially formed their councils, as part of the transition to the Jupyter governance model in December 2022.
```
Comment (afshin) [inline bootstrapping_subproject_councils.md:4]:
```suggestion
This document outlines how Subprojects initially formed their councils, as part of the transition to the Jupyter governance model in December 2022.
```

**Comment by afshin** [comment]
Thank you Jason for reorganizing and making these documents consistent!

Ready for review. We should complete the leadership section when that is possible – before merging.

File: people.md [hunk 2]
```diff
@@ -1,93 +1,84 @@
...
+-
+-
+-
+
+## [Software Steering Council](software_steering_council.md)
+
+| Subproject | Representative | GitHub username |
+| ---------- | -------------- | --------------- |
+| Jupyter Accessibility | Isabela Presedo-Floyed | [@isabela-pf](https://github.com/isabela-pf) |
+| Jupyter Foundations | | |
```
Comment (jasongrout) [inline people.md:20]: We just finished the Foundations SSC election, and Paul was chosen as the representative.

```suggestion
| Jupyter Foundations | Paul Ivanov | [@ivanov](https://github.com/ivanov) |
```

**Comment by jasongrout** [comment]
I also started a PR in the website repo: https://github.com/jupyter/jupyter.github.io/pull/712

**Review by fperez — APPROVED** [review]

**Pull request merged**